docs/sidebars.ts

```diff
@@ -102,6 +102,7 @@ const sidebars: SidebarsConfig = {
           items: [
             "contribution/dev-cli",
             "contribution/tools/debugging",
+            "contribution/tools/ethspecify",
```

Comment:
can we put this at the of items

Reply:
we have an array of `items` here, I would prefer we add this to the end of the list, instead of the middle